datastore/sampling/cross_validation.py

```diff
@@ -25,6 +25,9 @@ def stratified_split(dataset, num_splits, seed=42):
     skf = StratifiedKFold(n_splits=num_splits, random_state=seed)
     data, labels = dataset.load_data()
 
+    if label:
+        labels = labels[label]
```

Comment:
This seems too brittle. Users would assume that labels returned by `InMemoryDatasets` will be dictionaries, and assumes that the user will pass the appropriate `label` key. 

This might warrant creating a separate function, `multitask_stratified_split` to make thing more explicit, even at the expense of code duplication.

Reply:
Decided to create a separate function, `multitask_stratified_split`. It is a bit of code duplication, but I think less bug prone than trying to do it all in `stratified_split`. I can imagine that future iterations of `multitask_stratified_split` could do fancier multi-label stratification.